docs/tutorial/chapter-3-adding-mutations-to-your-api.md

```diff
@@ -256,7 +256,7 @@ type Post {
 
 </div>
 
-Evidently most of data isn't nullable! Its pretty annoying to write and read that repitition. Luckily Nexus allows us to change the nullability defaults at the object level.
```

Comment:
Do you have a vscode spell check workflow to suggest?

Reply:
I don't use any :/. Jetbrains all have spell check built-in, even within your code and it's great. This one might be worth testing https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker ?